test/cookie.js

```diff
@@ -43,6 +43,16 @@ describe('new Cookie(name, value, [options])', function () {
         var cookie = new cookies.Cookie('foo', 'bar', { maxage: 86400 })
         assert.equal(cookie.maxage, 86400)
       })
+
+      it('should not set max-age attribute', function () {
+        var cookie = new cookies.Cookie('foo', 'bar', { maxage: 86400 })
+        assert.equal(/; max-age=/.test(cookie.toHeader()), false)
```

Comment:
The tests in this file should be testing the underlying properties of the cookie object for the most part, where the other tests (really just `test.js` now) testing the serialized header. You can test the header here too if you like, but at least you'll want to test the populated properties 👍